nvm.sh

```diff
@@ -1999,6 +2031,10 @@ nvm_supports_source_options() {
   [ "_$(echo 'echo $1' | . /dev/stdin yes 2> /dev/null)" = "_yes" ]
 }
 
+nvm_supports_xz() {
+  [ $(command which xz) ] && nvm_is_merged_node_version "$1"
```

Comment:
instead of a subshell, could this just be `command which xz 2>&1 >/dev/null && nvm_is_merged_node_version "$1"`?